pkg/account/storage/v2/admin_account.go

```diff
@@ -34,7 +34,7 @@ var (
 	selectAdminAccountV2SQL string
```

Comment:
How about renaming the SQL file and the constant, too?

```suggestion
        //go:embed sql/account_v2/select_system_admin_account_v2.sql
        selectSystemAdminAccountV2SQL string
```

Reply:
I'm fixing in the next PR. There are some other places to be fixed too.